lib/galaxy_test/selenium/test_sign_out.py

```diff
@@ -0,0 +1,22 @@
+from .framework import (
+    selenium_test,
+    SeleniumTestCase
+)
+
+
+class SignOutTestCase(SeleniumTestCase):
+    @selenium_test
+    def test_sign_out(self):
+        email = self._get_random_email()
+        self.register(email)
+        self.click_masthead_user()
+        self.components.masthead.preferences.wait_for_and_click()
+        self.components.preferences.sign_out.wait_for_and_click()
+        cancel_button = self.driver.find_element_by_xpath("//button[@id='button-0']")
```

Comment:
Very nice test, especially ```assert not self.is_logged_in()``` very cool. I have just a minor notice. Can you make something similar to ```libraries.folder.import_datasets_ok_button``` in ```navigation.yaml``` instead of xpath here, please?